examples/minimal.py

```diff
@@ -0,0 +1,14 @@
+import bayem
```

Comment:
General:
As a principal component of the VB is the Jacobian, IMO, it makes perfect sense to provide a minimal example with a user-given Jacobian as well.